.github/workflows/fondue-pull-requests.yaml

```diff
@@ -148,8 +148,19 @@ jobs:
                   node-version-file: ".nvmrc"
                   cache: "pnpm"
 
+            - name: Cache Cypress binary
+              uses: actions/cache@v3
+              with:
+                  path: ~/.cache/Cypress
+                  key: cypress-13.6.6-${{ runner.os }}-cypress-${{ hashFiles('**/pnpm-lock.yaml')}}
+                  restore-keys: |
+                      cypress-13.6.6-${{ runner.os }}-cypress-
+
```

Comment:
I have noticed the "old-component" jobs are building a cache for the `node_modules` that persists across workflow (and the cypress cache from those jobs is also handled within the same action).

Right now in this current workflow (`Components`) we are not doing that, so I think it's best to keep the cache and actions separated.